### ISSUE TITLE
I haven't been able to make Alex work since around 6.3-6.4

### ISSUE THREAD

**Issue by NargiKyrad** [Soreepeong/XivAlexander#691]
I swapped to No Clippy a while back due to this, but it's annoying having to wait for Clippy to be back alongside dalamud plugins. 
I tried the opcode for 6.58 but it's also not working for me

{
	"C2S_ActionRequest": "0x017f",
	"C2S_ActionRequestGroundTargeted": "0x02c7",
	"Common_UseOodleTcp": true,
	"PatchCode": [],
	"S2C_ActionEffect01": "0x00d3",
	"S2C_ActionEffect08": "0x0356",
	"S2C_ActionEffect16": "0x021e",
	"S2C_ActionEffect24": "0x0092",
	"S2C_ActionEffect32": "0x0237",
	"S2C_ActorCast": "0x0165",
	"S2C_ActorControl": "0x02c9",
	"S2C_ActorControlSelf": "0x0340",
	"Server_IpRange": "119.252.37.0/24, 119.252.36.0/24, 153.254.80.0/24, 204.2.29.0/24, 80.239.145.0/24",
	"Server_PortRange": "1-65535"
}

This is the Oopcode i have. someone suggested a fix of changing the values on Server_IpRange but that also did not work

**Comment by madara4567** [comment]
yeah xivalex is bs, shit takes even longer than noclippy to get working

**Comment by NargiKyrad** [comment]
They update the opcodes regularly and i've tried re-installing it from scratch as well, i just can't make it work even with the current ones
